tfx/experimental/pipeline_testing/examples/chicago_taxi_pipeline/taxi_pipeline_dummy_e2e_test.py

```diff
@@ -0,0 +1,164 @@
+# Lint as: python2, python3
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""E2E Tests for tfx.examples.chicago_taxi_pipeline.taxi_pipeline_beam."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import absl
+import os
+from typing import Text
+
+import tensorflow as tf
+
+from tfx.examples.chicago_taxi_pipeline import taxi_pipeline_beam
+from tfx.experimental.pipeline_testing import dummy_component_launcher
+from tfx.orchestration import metadata
+from tfx.orchestration.beam.beam_dag_runner import BeamDagRunner
+from tfx.orchestration.config import pipeline_config
+
+class TaxiPipelineBeamEndToEndTest(tf.test.TestCase):
```

Comment:
Can you change the name of the class to match the filename? It might confuse users while reading logs.